v2/utils.go

```diff
@@ -216,3 +218,24 @@ func ToResources(spec *specs.LinuxResources) *Resources {
 	}
 	return &resources
 }
+
+// Gets uint64 parsed content of single value cgroup stat file
+func getStatFileContent(filePath string) uint64 {
```

Comment:
nit: function name is not specific, but it only can retrieve uint64 value.

Reply:
Also, this should take `io.Reader` rather than a filepath string and should have unit tests. Can be a follow-up PR.

Reply:
Yep, i have in plans to cover everything with unit tests but in separate PR's

Reply:
Also, I think that `io.Reader` parameter here will over complicate using of that function.